src/manager.cpp

```diff
@@ -518,4 +527,39 @@ types::ListOfPaths Manager::getFrusByExpandedLocationCode(
 
 void Manager::performVPDRecollection() {}
 
+void Manager::performBackupAndRestore()
```

Comment:
I don't understand the need of this API.
Why manager needs to have this intelligence at all. Manager need to call back up and restore and the class need to take care of that.

Reply:
When the system comes up very first time & one of the source or destination is of cache scenario was triggering backup & restore. So added this API to avoid it.

As we discussed, will remove this API. To keep generic, backup & restore will be triggered at the end when all VPD's are collected (if not called earlier). 

Removing this API.